test/unittests/typed_value_test.cpp

```diff
@@ -0,0 +1,29 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2020 The Fizzy Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "test/utils/typed_value.hpp"
+#include <gtest/gtest.h>
+
+using namespace fizzy::test;
+
+TEST(typed_value, u32_literal)
+{
+    static_assert(std::is_same_v<decltype(0_u32), uint32_t>);
+    EXPECT_EQ(0_u32, uint32_t{0});
+    EXPECT_EQ(1_u32, uint32_t{1});
+    EXPECT_EQ(0xffffffff_u32, uint32_t{0xffffffff});
+
+    unsigned long long input = 0x100000000;  // NOLINT
```

Comment:
Why NOLINT is needed?

Reply:
CLion says I should be using `uint64_t`.